src/models/free-busy.ts

```diff
@@ -4,7 +4,34 @@ import Attributes from './attributes';
 export type FreeBusyQuery = {
   startTime: number;
   endTime: number;
-  emails: string[];
+  emails?: string[];
+  calendars?: FreeBusyCalendarsProperties[];
+};
+
+export type FreeBusyCalendarsProperties = {
+  accountId: string;
+  calendarIds: string[];
+};
+
+export class FreeBusyCalendars extends Model
+  implements FreeBusyCalendarsProperties {
+  accountId = '';
+  calendarIds: string[] = [];
+
+  constructor(props?: FreeBusyCalendarsProperties) {
+    super();
+    this.initAttributes(props);
+  }
+}
+FreeBusyCalendars.attributes = {
+  accountId: Attributes.String({
+    modelKey: 'accountId',
+    jsonKey: 'account_id',
+  }),
+  calendarIds: Attributes.StringList({
+    modelKey: 'calendarIds',
+    jsonKey: 'calendar_ids',
+  }),
 };
```

Comment:
It is probably cleaner to assign it inside the class definition.
```suggestion
export class FreeBusyCalendars extends Model
  implements FreeBusyCalendarsProperties {
  accountId = '';
  calendarIds: string[] = [];
  static attributes: Record<string, Attribute> = {
    accountId: Attributes.String({
      modelKey: 'accountId',
      jsonKey: 'account_id',
    }),
    calendarIds: Attributes.StringList({
      modelKey: 'calendarIds',
      jsonKey: 'calendar_ids',
    }),
  };

  constructor(props?: FreeBusyCalendarsProperties) {
    super();
    this.initAttributes(props);
  }
}
```

Reply:
This is a really good idea. Thanks for picking this out! I'm not going to commit this change in this PR because _all_ classes are like this, so I'd rather make another PR where we change all the classes to be like this.